components/clm/bld/namelist_files/namelist_defaults_clm4_5.xml

```diff
@@ -213,6 +213,10 @@ ic_tod="0" sim_year="2000" glc_nec="0" use_crop=".true." irrigate=".true." nu_co
 <finidat hgrid="ne0np4_enax4v1" sim_year="2000">lnd/clm2/initdata_map/clmi.ICRUCLM45SP.2000-01-01.enax4v1_oRRS18to6_simyr2000_c180430.nc</finidat>
 <finidat hgrid="ne0np4_twpx4v1" ic_ymd="101" sim_year="2000">lnd/clm2/initdata_map/clmi.ICRUCLM45SP.2000-01-01.twpx4v1_oRRS18to6v3_simyr2000_c180430.nc</finidat>
 
+<!-- for present day simulations - year 2010 -->
+<fsurdat hgrid="r05"   sim_year="2010" use_crop=".false." >
+	lnd/clm2/surfdata_map/surfdata_0.5x0.5_simyr2000_c190418.nc</fsurdat>
```

Comment:
It seems like a surfdata_0.5x0.5_simyr2010 file is better. Is that possible to obtain that file?

Reply:
If the surfdata_0.5x0.5_simyr2010 file is too hard to get, then the simyr2000 file can be a temporary solution.

Reply:
Ideally, yes. I can't find one for r05 and 2100 on inputdata server nor cori inputdata path.The 2000 one is current default for F2100 and grid like ne30_r05_oECv3. Just to  ensure it will work the same way after merging #3449 for now.

Reply:
Correction: We do have one for 2010 on inputdata server
 surfdata_0.5x0.5_simyr2010_c191025.nc